provisioner.go

```diff
@@ -103,19 +104,23 @@ func (p *defaultProvisioner) Provision(ctx context.Context, d Dispatcher, appNam
 	}
 	s.Astilectron = &ProvisionStatusPackage{Version: VersionAstilectron}
 
+	if s.Electron == nil {
+		s.Electron = make(map[string]*ProvisionStatusPackage)
+	}
 	// Provision electron
-	if err = p.provisionElectron(ctx, d, paths, s, appName, os); err != nil {
+	if err = p.provisionElectron(ctx, d, paths, s, appName, os, arch); err != nil {
 		err = errors.Wrap(err, "provisioning electron failed")
 		return
 	}
-	s.Electron = &ProvisionStatusPackage{Version: VersionElectron}
+
+	s.Electron[fmt.Sprintf("%s-%s", os, arch)] = &ProvisionStatusPackage{Version: VersionElectron}
```

Comment:
The following code is used twice:

```go
fmt.Sprintf("%s-%s", os, arch)
```

I'd rather see it in a `statusElectronKey` function.